src/main/java/com/minh/project/configuration/CORSFilter.java

```diff
@@ -0,0 +1,29 @@
+package com.minh.project.configuration;
+
+import java.io.IOException;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletResponse;
+
+
+public class CORSFilter implements Filter {
```

Comment:
So you don't really need the CORSfilter right now since your app wont be pinged by a different app the filter is just a pass through right now and even if there was another app that could ping your app directly there is a better way to do what the CORSfilter does.